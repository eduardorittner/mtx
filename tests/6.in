this is a te
st
of whether deleting a newline character works
