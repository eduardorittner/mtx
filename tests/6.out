this is a test
of whether deleting a newline character works
